lib/bolt/node/winrm.rb

```diff
@@ -9,40 +9,52 @@ def protocol
       'winrm'
     end
 
-    def initialize(host, port, user, password, shell: :powershell, **kwargs)
-      super(host, port, user, password, **kwargs)
+    def port
+      default_port = @insecure ? 5985 : 5986
+      @port || default_port
+    end
 
-      @shell = shell
```

Comment:
We didn't expose a way to get shell into the config before?

Reply:
Not as far as I can tell. The `shell` argument to the initializer is a separate arg from the config, not a part of it. Also, we immediately execute powershell code upon making the connection (`$PSVersionTable.PSVersion`), so I don't see how you could reasonably use another shell anyway.